### ISSUE TITLE
ReferenceUsedNamesOnly: importing global functions and constants should not be necessary when in the global namespace

### ISSUE THREAD

**Issue by nicwortel** [slevomat/coding-standard#276]
# Steps to reproduce

1. Create a file `index.php`:

```php
<?php

echo PHP_VERSION;

echo count([]);
```
2. Create a `phpcs.xml`:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<ruleset>
    <rule ref="SlevomatCodingStandard.Namespaces.ReferenceUsedNamesOnly" >
        <properties>
            <property name="allowFallbackGlobalFunctions" value="false"/>
            <property name="allowFallbackGlobalConstants" value="false"/>
        </properties>
    </rule>
</ruleset>
```
3. Run PHPCS:
```bash
vendor/bin/phpcs index.php
```

# Expected result

PHPCS passes: since we are in the global namespace, we shouldn't have to import functions or constants.

# Actual result
PHPCS reports errors:
```
--------------------------------------------------------------------------------------------------------------------
FOUND 2 ERRORS AFFECTING 2 LINES
--------------------------------------------------------------------------------------------------------------------
 3 | ERROR | [x] Constant PHP_VERSION should not be referenced via a fallback global name, but via a use statement.
 5 | ERROR | [x] Function count() should not be referenced via a fallback global name, but via a use statement.
--------------------------------------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 2 MARKED SNIFF VIOLATIONS AUTOMATICALLY
--------------------------------------------------------------------------------------------------------------------
```

Running `vendor/bin/phpcbf` will add the use statements, which PhpStorm will then mark as unnecessary:

![PhpStorm screenshot](https://user-images.githubusercontent.com/1055691/35974739-93c51502-0cda-11e8-9737-cf2cab2495e9.png)

(also note the fact that the use statements are for some reason inserted after the first statement)

`index.php` after running `vendor/bin/phpcbf`:

```php
<?php

echo PHP_VERSION;
use const PHP_VERSION;
use function count;

echo count([]);
```

**Comment by nicwortel** [comment]
Note that the sniff already seems to detect this for classes, as PHPCS reports no error for the following file:

```php
<?php

$dateTime = new DateTime();
```

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/ec1fd10ec2d2223b345f107d89039c7be5f8ceba

**Comment by nicwortel** [comment]
@kukulich thank you for the fix! Could you release/tag a new version containing this fix?

**Comment by nicwortel** [comment]
I'm sorry, I was confused by the commit ec1fd10 only listed as part of the master branch and not as part of a tag, but it appears that the fix is actually in the current release. So ignore my previous message 😉
